dot/core/interface.go

```diff
@@ -58,7 +60,8 @@ type StorageState interface {
 	LoadCode(root *common.Hash) ([]byte, error)
 	LoadCodeHash(root *common.Hash) (common.Hash, error)
 	TrieState(root *common.Hash) (*rtstorage.TrieState, error)
-	StoreTrie(*rtstorage.TrieState, *types.Header) error
+	//StoreTrie(*rtstorage.TrieState, *types.Header) error
+	StoreTrieVdt(*rtstorage.TrieState, *types.HeaderVdt) error
```

Comment:
Are you going to replace `type.Header` and revert to the original function names?

Reply:
Yup exactly. Anything with "new" or "vdt" appended to the name is temporary and will be changed back to the original naming after the integration.

Reply:
can you do the renames I think it'll make the diff look much nicer.

Reply:
Absolutely I agree. I am actually working on this currently